### ISSUE TITLE
VRAM low usage

### ISSUE THREAD

**Issue by MacZarFx** [madMAx43v3r/chia-gigahorse#254]
I use rtx3090 for plotting - Gigahorses uses only 8 out of 24gb VRAM. Is there any parameter/method to push plotter for full VRAM utilization?
I have 128GB RAM onboard and using -M64 parameter;  -t  and -2 fast NVME drives.
What should I change?

**Comment by LimakKAAM** [comment]
Use  approx -M 12 - 16 graphic memory.

**Comment by MacZarFx** [comment]
Unfortunatelly nothing has change in plus. The lower -M parameter is the lower usage of GPU card is :( And what interesting plotting time is nearly the same - the first plot k32 C19 in approx. 2.5 min...
I hoped the more VRAM on GPU I have the faster plooting will be - but I can't find a proper way to use all available GPU VRAM... :(

ps. Maybe I should use -B --chunksize arg or -S   --streams arg? But I can't find any detailed explanation what it is about and if (and how) it can change the plotting time...

**Comment by Mathis436** [comment]
you are correct.... -S for the streams argument will work. I believe using a -S 12 should max out your vram, although you wont see much difference in the plot time in my experience. Max did a great job in optimizing the plotter.

**Comment by MacZarFx** [comment]
Thanks a lot!
